e2e/wdio/platform/pages/button.po.ts

```diff
@@ -0,0 +1,30 @@
+import { BaseComponentPo } from './base-component.po';
+import { waitForPresent } from '../../driver/wdio';
+
+export class ButtonPo extends BaseComponentPo {
+    private url = '/menu-button';
+    root = '#page-content';
```

Comment:
remove this line, not needed here cz this variable already present in base-component.po and will be taken from there